dascore/core/inventory.py

```
@@ -0,0 +1,1600 @@
+        """
+        lo = self.start_distance if distance[0] is None else float(distance[0])
+        hi = self.end_distance if distance[1] is None else float(distance[1])
+        lo = max(lo, self.start_distance)
+        hi = min(hi, self.end_distance)
+        if hi <= lo:
+            msg = f"Empty distance selection ({distance})."
+            raise ParameterError(msg)
+        components = []
+        for comp, (c_lo, c_hi) in zip(
+            self.optical_components, self.component_intervals()
+        ):
```

Ruff (0.16.1)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)